packages/truffle-box/lib/utils/unbox.js

```diff
@@ -135,23 +119,22 @@ async function copyTempIntoDestination(tmpDir, destination, options) {
   }
 
   for (const file of shouldCopy) {
-    fs.copySync(`${tmpDir}/${file}`, `${destination}/${file}`);
+    fse.copySync(`${tmpDir}/${file}`, `${destination}/${file}`);
   }
 }
 
-function installBoxDependencies(boxConfig, destination) {
-  var postUnpack = boxConfig.hooks["post-unpack"];
+function installBoxDependencies({ hooks }, destination) {
+  const postUnpack = hooks["post-unpack"];
 
-  return new Promise(function(accept, reject) {
-    if (postUnpack.length === 0) {
-      return accept();
-    }
+  if (postUnpack.length === 0) {
+    return;
+  }
 
-    exec(postUnpack, { cwd: destination }, function(err, stdout, stderr) {
-      if (err) return reject(err);
-      accept(stdout, stderr);
-    });
-  });
+  try {
+    execSync(postUnpack, { cwd: destination });
+  } catch (error) {
+    throw error;
+  }
```

Comment:
We can get rid of this `try`/`catch` because we don't anything with the exception ourselves